prisma/schema.prisma

```diff
@@ -329,3 +331,10 @@ enum therapy {
   ART
   OTHER
 }
+
+enum programTags {
+  MUSIC
+  ARTS
+  VIRTUAL_RELAITY
+  DRAMA
+}
```

Comment:
If you want to change the prisma file, you'll also need to change the `schema.sql` file to reflect these changes. Otherwise the database doesn't actually have these columns atm.